docs/about/babe-explained.md

```diff
@@ -0,0 +1,75 @@
+---
+id: babe
+title: BABE
+sidebar_label: BABE
+description: Overview of the Avail usage of BABE
+keywords:
+  - docs
+  - avail
+  - BABE
+  - concensus
+image: https://docs.availproject.org/img/avail/AvailDocs.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Introduction
+
+Blind Assignment for Blockchain Extension (BABE) serves as a block production engine, drawing inspiration from Ouroboros Praos, another proof-of-stake protocol. It functions
+autonomously, offering probabilistic finality, or it can be integrated with a finality mechanism like GRANDPA.
+
+BABE operates on a slot-based algorithm, dividing time into eras, epochs, further segmented into slots. Within the Avail context, each slot lasts for twenty seconds, aligning with the
+target block time. In every slot, BABE determines an author (or potentially multiple authors) responsible for producing a block.
+
+On Avail's explorer, you can observe the eras and epochs by navigating to the staking page and locating the information at the top-right corner.
+
+<img src={useBaseUrl("img/avail/epoch-era.png")} width="100%" height="100%" />
+
+## BABE Block Production Process
+
+Every slot has the potential for both a primary and secondary author, also known as a "slot leader." These primary and secondary authors are also known as validatrs. The assignment of
+primary slot leaders is a random process. However, due to the inherent randomness, there are instances where slots may lack a designated leader. To maintain a consistent block time,
+BABE employs a round-robin system to designate secondary slot leaders.
```

Comment:
Not sure what you mean by "difference algorithm" or do you mean different algorithms? But validators are usually selected randomly to avoid prediction of next block producer (for security purposes), but if for some reason node can not do the work like offline, respond late I believe the secondary slot leader jumps in. From my understanding the secondary slot leader is determined by round robin.

Here is some references
https://wiki.polkadot.network/docs/learn-consensus
https://medium.com/polkadot-network/polkadot-consensus-part-3-babe-dcc2e0dd8878

Let me know if avail has changed this

Reply:
Nice. Found more info about this at this exact link:
[https://wiki.polkadot.network/docs/learn-consensus#no-validators-in-slot](https://wiki.polkadot.network/docs/learn-consensus#no-validators-in-slot)

I am changing the text slightly to this in a small PR where I make some more changes to make everything fit in with the rest of the docs:

"_Each slot may have both a primary and a secondary author, chosen from amongst a set of active validators. The assignment of
primary authors, aka slot leaders, is a random process governed by the VRF function. However, due to the way the function works, there may be instances where no validators qualify as the primary author. To maintain a consistent block time,
BABE employs a round-robin system to designate secondary slot leaders, who produce new blocks without a designated leader._"